config/hooks.go

```diff
@@ -7,8 +7,11 @@ import (
 // Hooks holds customizable functions that can be called at varying points by the daemon to.
 // integrate with other tools.
 type Hooks struct {
-	// OnBootstrap is run after the daemon is initialized and bootstrapped.
-	OnBootstrap func(s *state.State, initConfig map[string]string) error
+	// PreBootstrap is run before the daemon is initialized and bootstrapped.
```

Comment:
Does this break any compatibility with libraries that import `MicroCluster`? I guess the change would land right away in `main`.

Reply:
Yeah, they would all have to rename `OnBootstrap` to `PostBootstrap`. I suppose we should begin tagging versions soon, given both this change and the non-cluster member PR will introduce some changes.

Reply:
Ok thanks, I didn't check if there are any releases yet.